WordPress/Classes/ViewRelated/Blog/TimeZoneSelectorViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+
+class TimeZoneSelectorViewController: UITableViewController, ImmuTablePresenter {
+
+    /// users current timezone passed by SiteSettingsVC, if empty means manual offset is to be used
```

Comment:
Sorry to nitpick, but if these are written as doc comments (triple `///` instead of `//`) they should use correct capitalization and punctuation and be written in complete sentences – as if they were documentation for Apple's own APIs.

For example:

```
/// The blog's current timezone. If no timezone string is set, then the manual offset value will be used.
```